seal/managers.py

```diff
@@ -24,4 +25,12 @@ def _clone(self, **kwargs):
         return clone
 
     def seal(self):
-        return self._clone(_sealed=True)
+        clone = self._clone(_sealed=True)
+        clone._prefetch_related_lookups = tuple(
+            models.Prefetch(
+                lookup,
+                self.model._meta.get_field(lookup).remote_field.model._default_manager.all(),
+            ) if isinstance(lookup, string_types) else lookup
```

Comment:
@bryanhelmig This still needs to handle nested string prefetches but that shouldn't be an issue (e.g. `previous_locations__previous_visitors`.